cfme/tests/infrastructure/test_customization_template.py

```diff
@@ -86,16 +88,19 @@ def test_duplicate_name_error_validation(collection):
     collection.delete(False, template_name)
 
 
-@pytest.mark.xfail(message='http://cfme-tests.readthedocs.org/guides/gotchas.html#'
-                           'selenium-is-not-clicking-on-the-element-it-says-it-is')
 def test_name_max_character_validation(collection):
-    """Test to validate name with maximum characters in customization templates."""
-
-    with error.expected('Name is required'):
-        template_name = collection.create(
-            name=fauxfactory.gen_alphanumeric(256),
-            description=fauxfactory.gen_alphanumeric(16),
-            image_type='RHEL-6',
-            script_type='Kickstart',
-            script_data='Testing the script')
-    collection.delete(False, template_name)
+    """Test to validate name with maximum characters in customization templates.
+       Max length is controlled by UI elements - we are not allowed to input more than we should
+       Opens template details to verify that extra symbols were cut
+    """
+    template_name = collection.create(
```

Comment:
**REQUIRED**: The collection should instantiate and return the object. Not just a name. I'm not sure why we need to create a second object here.

Reply:
@psav  : we create an object in UI. but UI cuts template name to 255 symbols only. so for convenience - I instantiate another object with correct name. otherwise it would be view.sidebar.tree.click_path() + no entity + delete clicking UI elements as we.
I'd rather remove this test at all =) - max length is under JS control right now

Reply:
so instead of recreating the object, just reassign the name?
```template_name.name = template_name[:255]``` after the create?

Reply:
@psav done